js/src/admin/components/MailPage.js

```diff
@@ -163,34 +162,55 @@ export default class MailPage extends Page {
     return this.fields.some(key => this.values[key]() !== app.data.settings[key]);
   }
 
+  sendingTestEmail() {
+    return this.sendingTest;
+  }
+
```

Comment:
We use Mithril props for this. `this.sendingTest = m.prop(false);`, then update it with `this.sendingTest(true)`.